chaoslib/litmus/container_kill/containerd_chaos/crictl-chaos.yml

```diff
@@ -1,5 +1,32 @@
 ---
 - block:
+    - block: 
+
+      - name: Select the app pod
```

Comment:
Note: FYI, We are putting 4 space in the block

Reply:
Done

Reply:
Yes, but inside the block we should indent